integration/spark/shared/src/main/java/io/openlineage/spark/agent/lifecycle/plan/SaveIntoDataSourceCommandVisitor.java

```diff
@@ -143,6 +146,21 @@ public List<OutputDataset> apply(SparkListenerEvent event, SaveIntoDataSourceCom
       }
     }
 
+    if (command
+        .dataSource()
+        .getClass()
+        .getCanonicalName()
+        .equals(JdbcRelationProvider.class.getCanonicalName())) {
+      if (command.options().contains("dbtable") && command.options().contains("url")) {
```

Comment:
is it possible that options are null ?

Reply:
it's not, JDBCOptions class won't let it happen